README.md

```diff
@@ -120,6 +117,45 @@ curl localhost:8080
 # Output: Hello World from PHP HTTP function!
 ```
 
+# Run your function in Cloud Run
+
+First you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
+[Google Cloud Project][gcp-project] you want to use.
+
+Next, build the container using the example `Dockerfile`:
+
+```sh
+docker build . \
+    -f vendor/google/cloud-functions-framework/examples/hello/Dockerfile \
+    -t gcr.io/$GCLOUD_PROJECT/my-cloud-function
+```
+
+> **NOTE**: Be sure to replace `$GCLOUD_PROJECT` with your Google Cloud project ID.
```

Comment:
Often our instructions have them set that first, like:
```
export PROJECT_ID=$(gcloud config get-value core/project)
```

Reply:
I personally have never seen this, can you point to an example?

Our docs usually have a `<var>` placeholder, and I've never done this for any PHP readmes.

Reply:
Yeah, it's more of the community tutorials that use the easy approach.

Reply:
I added it to the notes. I think it'd be better for them to explicitly set it, since it's possible they haven't called `gcloud init` yet, or they may be in a different project than the one they want to deploy.